test/cpp/run_tests.sh

```diff
@@ -39,35 +37,38 @@ do
       BUILD_ONLY=1
       ;;
     F)
-      FILTER="--gtest_filter=$OPTARG"
+      FILTER="--test_filter=$OPTARG"
```

Comment:
Does `--gtest_filter` still work or not?

Reply:
The filter works here when passed with `-F`. The correct flag is `--test_filter` in bazel, it passes it down to the xUnit (e.g. `googletest`) correctly. Tests can be manually filtered in bazel simply as `bazel test --test_filter=TensorTest.* //`. You could also pass a `test_arg` to it, circumventing bazel and just interacting with the C++ binary that is the test, like `bazel test --test_arg=--gtest_filter=TensorTest.* //`. I prefer the first mechanism but both will work.